pkg/image/admission/imagepolicy/api/v1/defaults.go

```diff
@@ -14,6 +14,20 @@ func SetDefaults_ImagePolicyConfig(obj *ImagePolicyConfig) {
 		obj.ResolveImages = Attempt
 	}
 
+	if obj.ResolutionRules == nil {
+		obj.ResolutionRules = []ImageResolutionPolicyRule{
+			{TargetResource: GroupResource{Resource: "pods"}, LocalNames: true},
+			{TargetResource: GroupResource{Resource: "builds"}, LocalNames: true},
```

Comment:
You can create a build directly with `From: {Kind: "DockerImage", Name: "mysql"}`.  This makes it consistent with pods.